### ISSUE TITLE
summary function from regressor package is giving valueError

### ISSUE THREAD

**Issue by uthambathoju** [nsh87/regressors#20]
I'm running a sklearn logistic regression model.when i try to get summary stats by using below line of code.I'm getting the following error.kindly help me on this.

```
from regressors import stats
log_mod=LogisticRegression(random_state=123)
model=log_mod.fit(X_train,y_train)
stats.summary(model, X_train ,y_train)

```
Error:
```
   251     )
    252     coef_df['Estimate'] = np.concatenate(
--> 253         (np.round(np.array([clf.intercept_]), 6), np.round((clf.coef_), 6)))
    254     coef_df['Std. Error'] = np.round(coef_se(clf, X, y), 6)
    255     coef_df['t value'] = np.round(coef_tval(clf, X, y), 4)

ValueError: all the input array dimensions except for the concatenation axis must match exactly

```

**Comment by Colonel-Meowzer** [comment]
This fix is fairly straightforward. Line 252-3 should be as follows
```python
coef_df['Estimate'] = np.concatenate(
(np.round(np.array([clf.intercept_]), 6), np.round((clf.coef_), 6)), axis = 1)
```

The only commits for this repo appear  to be on Dec 8, 2015 so it looks like a pet project that is no longer being maintained.

**Comment by nsh87** [comment]
Thanks @Colonel-Meowzer, that's mostly correct. The math should hold true still, of course, however I'm giving more focus to language modeling and CNNs/ResNets for regression results these days and don't have much time to keep the code up with new Python versions. Lots of models are getting some "explainability" methods lately, so I typically don't do OLS or similar anymore, so I'm not using this package much.

**Comment by nsh87** [comment]
Should be fixed in 10b88343f44c8525ad5411d531bae57e052deb9d now
